BAN-139: Tooltip: Auto-position

chavlji opened the pull request:
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/BAN-139)

#### Description
As per JIRA ticket

#### ChangeLOG

- [ ] I have added all notable deployment/development environment (onprem!) changes to [CHANGELOG.md](https://github.com/reciprocity/zen-ui/blob/main/README.md)


#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

mihaelpalfi reviewed (commented):
Nice work! Much more clean code and a lot more options. Just some minor things i noticed : 
1. When setting height the arrow is not shown anymore.
2. In the story with slotted content the tooltip is displayed on the bottom and is not visible.
3. On position left and right the arrow is not positioned correctly.
4. When i scroll to playground and back up the tooltips for the variant story change and are shown on the bottom and position is a little offset for the middle one.

chavlji commented:
@mihaelpalfi 
1. No way to change that (parent has overflow: auto/hidden so that it's scrollable). But the good thing is that designers don't want arrow with scrollable tooltips:
![Screenshot 2020-12-23 at 13 00 27](https://user-images.githubusercontent.com/5729421/102994181-ddf3a600-451e-11eb-831f-930d8843648f.jpg)
2. Fixed
3. Fixed
4. Fixed

f-jauregui reviewed (approved):
Awesome work! 🤚
